.github/workflows/get-next-meeting-date.yml

```diff
@@ -0,0 +1,60 @@
+name: Get Next Meeting Date
+on:
+  push:
+    branches: ["scrape-next-meeting-date"] # Runs on pushes to the scrape-next-meeting-date branch
+
+  schedule:
+    - cron: "0 0 * * 0" # Runs at midnight every Sunday
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets the GITHUB_TOKEN permissions to allow committing changes, so we can update the next meeting date
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  execute-script:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          repository: "hailexiao/boston-liquor-license-tracker"
```

Comment:
```suggestion
          repository: "codeforboston/boston-liquor-license-tracker"
```

Reply:
Yep, was planning to do that--just wanted to allow reviewers to see that the action actually ran in my repo (since it won't run in main until merged).